src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/carts.yml

```diff
@@ -35,7 +35,7 @@ admin_carts_create:
     expose: true
 
 admin_carts_edit_address:
-  path: /{cartId}/edit-address
+  path: /{cartId}/address
```

Comment:
Shouldn't it be addresses? Or `delivery-address` and `invoice-address` separately.

Reply:
+1